GraphQL/GraphQLHelper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using GraphQL;
```

Comment:
Please change the whole solution name and folders to AElf.GraphQL instead of GraphQL. You can reference AElf.Opentelemetry for consistency.